maths/triangular_numbers.py

```diff
@@ -0,0 +1,43 @@
+"""
+A triangular number or triangle number counts objects arranged in an
+equilateral triangle. This module provides a function to generate
+first `n` triangular numbers.
+
+For more information about triangular numbers, refer to:
+https://en.wikipedia.org/wiki/Triangular_number
+"""
+
+
+def generate_triangular_numbers(count: int) -> list[int]:
```

Comment:
Why generate the triangular numbers in a list? Why not just return the nth triangular number? The triangular numbers have a closed-form formula, after all: https://en.wikipedia.org/wiki/Triangular_number

Reply:
Okay, I'll modify the function to calculate nth triangular number